packages/next/package.json

```
@@ -32,9 +32,9 @@
   "// These are relative from within the dist/ folder": "",
   "sideEffects": false,
   "peerDependencies": {
-    "next": "10.x",
-    "react": "^16.6.0 || ^17",
-    "twind": ">=0.15.9 <2",
+    "next": "10.x || 11.x",
+    "react": "^17",
+    "twind": "^0",
```

```suggestion
    "react": "^16.6.0 || ^17",
    "twind": ">=0.15.9 <2",
```

I would like the keep the react (v16 is widely used) and twind (0.15.9 introduced the API we are using here) ranges as is.